docs/dietpi_tools/system_maintenance.md

```diff
@@ -80,6 +80,37 @@ dietpi-backup
 
     ![DietPi-Backup filter option screenshot](../assets/images/dietpi-backup_filter-option.jpg){: width="681" height="330" loading="lazy"}
 
+     Every (otherwise excluded) parent directory of an included directory needs to be included as well:
+     
+     Eg. backup homeassistant userdata & default user data location:
+     
+        + /mnt/dietpi_userdata/
+        + /mnt/dietpi_userdata/homeassistant/
+        - /mnt/dietpi_userdata/Media/
+        - /mnt/dietpi_userdata/Music/
+        - /mnt/dietpi_userdata/Pictures/
+        - /mnt/dietpi_userdata/Video/
+        - /mnt/dietpi_userdata/*
+        
+        - /mnt/*
+        - /media/
```

Comment:
This needs to be wrapped into code tags, otherwise it will show up as formatted list:
```suggestion
    ```diff
    + /mnt/dietpi_userdata/
    + /mnt/dietpi_userdata/homeassistant/
    - /mnt/dietpi_userdata/Media/
    - /mnt/dietpi_userdata/Music/
    - /mnt/dietpi_userdata/Pictures/
    - /mnt/dietpi_userdata/Video/
    - /mnt/dietpi_userdata/*
    
    - /mnt/*
    - /media/
    ```
```

Reply:
The wrapping is not needed due to the indentation mixpc did.
Without indentation you need the fences.

Reply:
Ah right. However, for syntax highlighting, consistency, and to see code blocks on first view, we should use fences everywhere.